README.md

```diff
@@ -44,7 +44,7 @@ The scripts and documentation in this project are released under the [MIT Licens
 
 <!-- references -->
 [pages]: https://pages.github.com
-[release-list]: /releases
+[release-list]: ../../releases
```

Comment:
Good catch!

Sadly, that's not 100% reliable either as the relative link won't match up correctly when viewing the README at, say, `https://github.com/actions/upload-pages-artifact/blob/main/README.md`. As unfortunate as it is, using the absolute URL is safest in this case. 👍🏻 

```suggestion
[release-list]: https://github.com/actions/upload-pages-artifact/releases
```

Reply:
I was going to complain and saw this comment. Thank you both!